### DISCUSSION TITLE
after installation on vps server centos 7.9 it works only on http but not https

### DISCUSSION THREAD

**Discussion by vitaliy-v** [directus/directus#4574] [Archive]
I installed directus 9, then I have access admin on http://api.website.com:8055
If I use https://api.website.com:8055 I got an error ERR_SSL_PROTOCOL_ERROR and nothing happens, website not found.
SSL certificate is installed properly.
---

### Project details
Directus Version: 9.0.0-rc.51
Environment: production
OS: Centos 7.9
Node: 14.16.0

**Comment by rijkvanzanten** [comment]
> SSL certificate is installed properly.

How did you install SSL in this case? 

What webserver do you use as a reverse proxy to add the domain to Directus?

Directus itself doesn't do any SSL certificate magic, so you'll have to add something like Nginx or Caddy in front to proxy the requests to Directus. See https://www.digitalocean.com/community/tutorials/how-to-set-up-a-node-js-application-for-production-on-ubuntu-18-04#step-4-—-setting-up-nginx-as-a-reverse-proxy-server for a good example

**Comment by vitaliy-v** [comment]
I'm using nginx for proxy requests.
I found that i'm using in nginx conf file https
proxy_pass   https://mywebsite:8055;
when I changed that to http.
so the right line should be  proxy_pass   http://mywebsite:8055;

I had as well another location that points files to another root folder,
like this one
    location ~*
^.+.(jpg|jpeg|gif|png|ico|zip|tgz|gz|rar|bz2|iso|doc|xls|exe|pdf|ppt|txt|tar|mid|midi|wav|bmp|rtf|mp3|ogv|ogg|flv|swf|mpeg|mpg|mpeg4|mp4|avi|wmv|js|css|3gp|sis|sisx|nth)$
{
        root /home/mywebsite/public_html/api/public;
        expires 30d;
        add_header Pragma public;
        add_header Cache-Control "public, must-revalidate,
proxy-revalidate";
    }

So I removed that block and then it works!
Maybe it can help to someone

On Wed, Mar 17, 2021 at 6:52 PM Rijk van Zanten ***@***.***>
wrote:

> SSL certificate is installed properly.
>
> How did you install SSL in this case?
>
> What webserver do you use as a reverse proxy to add the domain to Directus?
>
> Directus itself doesn't do any SSL certificate magic, so you'll have to
> add something like Nginx or Caddy in front to proxy the requests to
> Directus. See
> https://www.digitalocean.com/community/tutorials/how-to-set-up-a-node-js-application-for-production-on-ubuntu-18-04#step-4-—-setting-up-nginx-as-a-reverse-proxy-server
> for a good example
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/directus/directus/discussions/4574#discussioncomment-494619>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAWODKGXLOQPT5LSOJRJOTDTEDM3BANCNFSM4ZK6W7ZQ>
> .
>


-- 
Development www.wecolour.com <http://wecolour.com>
Phone: +38 063 438 90 53, viber, whatsapp, telegram
Phone: +7 931 534 33 43 whatsapp
Vitaly Matveychenko
